console/atest-ui/src/views/net.ts

```diff
@@ -467,15 +445,11 @@ function GetSecrets(callback: (d: any) => void, errHandle?: (e: any) => void | n
 function FunctionsQuery(filter: string,
   callback: (d: any) => void, errHandle?: (e: any) => void | null) {
   const requestOptions = {
-    method: 'POST',
     headers: {
       'X-Auth': getToken()
-    },
-    body: JSON.stringify({
-      name: filter
-    })
+    }
   }
-  fetch('/server.Runner/FunctionsQuery', requestOptions)
+  fetch(`/api/v1/functions?name=${filter}`, requestOptions)
```

Comment:
Can we standardize here? It's all about restful style and avoiding the plain way of passing parameters.

Reply:
Of course, I will change the API style in this PR.

Reply:
got it